pkgs/top-level/stage.nix

```diff
@@ -259,6 +259,11 @@ let
       overlays = [ (self': super': {
         pkgsStatic = super';
       })] ++ overlays;
+    } // {
+      crossSystem = {
+        isStatic = true;
+        parsed = stdenv.hostPlatform.parsed;
+      };
```

Comment:
Can we combine this with the case for Linux below?  It's a bit confusing to see the generic crossSystem defined, and then immediately be overridden on Linux.

Reply:
Yes, I meant more like this:

```nix
parsed = if stdenv.isLinux then makeMuslParsedPlatform stdenv.hostPlatform.parsed else stdenv.hostPlatform.parsed;
```

or

```nix
parsed = (if stdenv.isLinux then makeMuslParsedPlatform else lib.id) stdenv.hostPlatform.parsed;
```